resources/js/visual-editor/site-editor/entity-editor-canvas.css

```
@@ -0,0 +1,140 @@
+    font-size: 12px;
+    color: var(--ap-site-editor-accent, #1d4ed8);
+    font-weight: 600;
+}
+
+.ap-site-editor__dirty-indicator::before {
+    content: '';
+    display: inline-block;
+    width: 8px;
+    height: 8px;
+    border-radius: 50%;
+    background: currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)